lib/LessStringifier.js

```diff
@@ -24,7 +30,7 @@ module.exports = class LessStringifier extends Stringifier {
     if (node.nodes) {
       this.block(node, name + params + important);
     } else {
-      const end = (node.raws.between || '') + important + (semicolon ? ';' : '');
+      const end = important + (semicolon ? ';' : '') + (node.raws.between || '');
```

Comment:
I'm not 100% satisfied with this. There's one stringify test ("mixin with !important") that's failing without it so I thought I'd leave it in for the sake of discussion.

I haven't been able to find any downsides to it yet, though. But let me know what you think and we'll take it from there.